src/pages/new-chat/new-chat.html

```diff
@@ -10,9 +10,15 @@
 
 <ion-content class="new-chat">
   <ion-list class="users">
-    <button ion-item *ngFor="let user of users | async" class="user" (click)="addChat(user)">
+    <button ion-item *ngFor="let user of users | async | search:searchPattern:'profile.name'" class="user" (click)="addChat(user)">
```

Comment:
I see, it causes the view to flicker for some reason which doesn't look good, so I decided to drop this one.